tests/Feature/Http/Livewire/DelegateDataBoxesTest.php

```diff
@@ -222,3 +222,89 @@ function createRoundWithDelegatesAndPerformances(array $performances = null, boo
         ->call('pollStatistics')
         ->assertSee('4,234,212');
 });
+
+it('should calculate not forging correctly', function () {
+    createRoundWithDelegatesAndPerformances([true, true, true, true, true], true, 50);
+    createRoundWithDelegatesAndPerformances([false, false, false, false, false], false, 1);
+
+    (new NetworkCache())->setHeight(fn (): int => 4234212);
+
+    Livewire::test(DelegateDataBoxes::class)
+        ->call('setIsReady')
+        ->call('pollStatistics')
+        ->assertSeeInOrder([
+            'Forging',
+            50,
+            'Missed',
+            0,
+            'Not Forging',
+            1,
+            'Current Height',
+        ]);
+});
+
+it('should calculate missed correctly', function ($performances, $addBlockForNextRound) {
+    createRoundWithDelegatesAndPerformances([true, true, true, true, true], true, 50);
+    createRoundWithDelegatesAndPerformances($performances, $addBlockForNextRound, 1);
+
+    (new NetworkCache())->setHeight(fn (): int => 4234212);
+
+    Livewire::test(DelegateDataBoxes::class)
+        ->call('setIsReady')
+        ->call('pollStatistics')
+        ->assertSeeInOrder([
+            'Forging',
+            50,
+            'Missed',
+            1,
+            'Not Forging',
+            0,
+            'Current Height',
+        ]);
+})->with([
+    'missed, missed, missed, forged, missed' => [
```

Comment:
what about `'forged, missed, forged, missed, missed'` ? that should show "not forging: 1"

Reply:
added that for you